src/autolabel/transforms/base.py

```diff
@@ -71,7 +81,14 @@ async def apply(self, row: Dict[str, Any]) -> Dict[str, Any]:
                 # Cache hit
                 return output
 
-        output = await self._apply(row)
+        try:
+            output = await self._apply(row)
+            output[self.transform_status_column] = True
+        except Exception as e:
+            logger.error(f"Error applying transform {self.name()}. Exception: {e}")
```

Comment:
can we put the error message in applied successfully as well? Would be easier to know what the error was for a specific row along with seeing the logs.

Reply:
I think we should not store a boolean and string value in the same column. To store the error we might go with just transformation_error as a column instead of applied successfully. WDYT

Reply:
makes sense